pkgs/by-name/ka/kani/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  fetchpatch2,
+  makeBinaryWrapper,
+  patchelf,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "kani";
+  version = "0.66.0";
+
+  src = fetchCrate {
+    pname = "${pname}-verifier";
+    inherit version;
+    hash = "sha256-mBulUFutwdHhaUtpe9r96kLmF/ZouW7NPiqA9Gb5TyY=";
+  };
+  cargoHash = "sha256-HNkdsJh45YNujEsbwOB5ZBv7YH2d2eWRyUwhSAS+LNQ=";
+
+  patches = [
+    (fetchpatch2 {
+      name = "kani-stub-ld.patch";
+      url = "https://github.com/model-checking/kani/commit/76a4adc4de498b16b9ad3cf9ae77e53ed9c54c7a.patch";
+      hash = "sha256-hTkzfjlEWNEd0FshaR9+EVUEdblTk/weV0Ij7E/H9dQ=";
+    })
+  ];
+
+  nativeBuildInputs = [
+    makeBinaryWrapper
+  ];
+  buildInputs = [
+    patchelf
+  ];
+  preFixup = ''
+    for file in $out/bin/* ; do
+      wrapProgram $file --set NIX_CC $NIX_CC --prefix PATH : ${lib.makeBinPath [ patchelf ]}
+    done
+  '';
+
+  meta = rec {
+    description = "Kani Rust Verifier";
+    longDescription = "A bit-precise model checker for Rust";
```

```suggestion
    description = "A bit-precise model checker for Rust";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes